ir/src/main/scala/org/scalajs/core/ir/ScalaJSVersions.scala

```diff
@@ -10,7 +10,7 @@ object ScalaJSVersions {
    */
 
   /** Scala.js version. */
-  val current: String = "0.6.7"
+  val current: String = "0.6.8-SNAPSHOT"
```

Comment:
Yes. 0.6.7 is forward binary compatible with 0.6.6. It emits the same version of the IR.